### DISCUSSION TITLE
Which is the Scheme implementation/dialect used for Zed extensions?

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#9988] [Feature Requests]
I have no idea which Scheme implementations/dialects Zed uses. For example, Doom Emacs has the following flags for Scheme:
- +chez ::
- +chibi ::
- +chicken ::
- +gambit ::
- +gauche ::
- +guile ::
- +kawa ::
- +mit ::
- +racket ::

So, the number of possibilities is big. I'm a newbie to Scheme, so please bear with me.

**Comment by maxdeviant** [comment]
The `.scm` files we use for Tree-sitter queries are just S-expressions, not any particular Lisp/Scheme dialect.

We just use the `.scm` extension for syntax highlighting.
